File: src/main/java/com/tom/guess/Game.java
package com.tom.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        /*int i = 0;
        do{
            System.out.println(i);
            i++;
        }while (i < 3);*/

        Random random = new Random();
        int secret = random.nextInt(10)+1;
        System.out.println(secret);
        Scanner scanner = new Scanner(System.in);
        int number = 0;
        while (number != secret){
            System.out.print("Please enter your number:");
            number = scanner.nextInt();
            if (number > secret){
                System.out.println("lower");
            } else if (number < secret) {
                System.out.println("higher");
            }else {
                System.out.println("Great, the number is "+number);
            }
        }
    }
}

File: src/main/kotlin/com/tom/Game.kt
package com.tom

import java.util.Random
import java.util.Scanner

fun main() {
    val secret = Random().nextInt(10)+1 //(0~9)+1
    println(secret)
//    val scanner = Scanner(System.`in`)
    var number = 0
    while (number != secret){
        print("Please enter your number: ")
//        number = scanner.nextInt()
        number = readLine()!!.toInt()
        if (number > secret){
            println("lower")
        }else if (number < secret){
            println("higher")
        }else{
            println("Great, the number is $number")
        }
    }
}